examples/model_loader.py

```diff
@@ -0,0 +1,94 @@
+from yggdrasil.examples import yamls as ex_yamls
+import yaml
+import jupyterlab_nodeeditor as jlne
+
```

Comment:
Why is this in `examples` if it's the main model loading code?

Reply:
So now it's part of a module.  If you add the import to the `__init__.py` file, it will appear in the top level of that module.

Reply:
So something like this: `from .model_loader import load_model` in the file, correct?

Reply:
I believe so

Reply:
Cleaned it up, moved it to the core package